.github/workflows/deploy.yml

```diff
@@ -8,24 +8,40 @@ on:
   push:
     branches:
       - main
+  schedule:
+    # Run weekly at 00:00 on Sunday
+    - cron: '0 0 * * 0'
 
 jobs:
   deploy:
     runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
```

Comment:
I don't find the meaning of `-l` and `{0}`. Could you show me some links?

Reply:
1. First point in https://github.com/conda-incubator/setup-miniconda#important
2. https://github.com/conda-incubator/setup-miniconda#use-a-default-shell